src/IW/Core/PGArray'.hs

```diff
@@ -0,0 +1,22 @@
+{-# LANGUAGE DeriveAnyClass #-}
+
+module IW.Core.PGArray'
```

Comment:
`PGArray'` is a poor name because it doesn't reflect the purpose of this data type. I would propose to rename it to `SqlArray`. This name is more different from `PGArray` which signals that it has different semantics.

Reply:
@chshersh That's funny I actually had it as `SqlArray` originally but change it back. I agree `SqlArray` is better,